test/tools/d_do_test.d

```diff
@@ -264,6 +313,25 @@ bool findTestParameter(const ref EnvData envData, string file, string token, ref
     return true;
 }
 
+/**
+ * Read the multi-line test parameter `token` from the source code and joins
+ * multiple definitions into a single string.
+ *
+ * ---
+ * TEST_OUTPUT:
+ * --
```

Comment:
How to escape `---` inside of a code block?

Reply:
Done, allthough it does require `preview=markdown` (and isn't rendered correctly in code-d...)